openfecli/parameters/misc.py

```diff
@@ -0,0 +1,11 @@
+import click
+from plugcli.params import Option
+
+N_PROTOCOL_REPEATS = Option(
+    "--n-protocol-repeats",
+    type=click.INT,
+    help="Number of independent run(s) to be run per execution of a transformation using the `openfe quickrun` command.\n\n"
```

Comment:
```suggestion
    help="Number of independent repeats(s) to be run per execution of a transformation using the `openfe quickrun` command.\n\n"
```
I know explaining the thing by saying the thing isn't great, but I do think this is slightly clearer - `independent` has a weird meaning here so saying `independent run` can be confusing.